mysql/datadog_checks/mysql/mysql.py

```diff
@@ -345,6 +360,8 @@ def check(self, instance):
 
         with self._connect(host, port, mysql_sock, user, password, defaults_file, ssl, connect_timeout, tags) as db:
             try:
+                self._conn = db
```

Comment:
If its not going to have a similar lifetime as the object I'd prefer not having it as an attribute an passing db to `self._query_manager.execute`

Reply:
Ok, I see it's not that easy to change